src/darwin/Framework/CHIP/MTRCertificates.h

```diff
@@ -45,7 +50,14 @@ NS_ASSUME_NONNULL_BEGIN
 + (MTRCertificateDERBytes * _Nullable)createRootCertificate:(id<MTRKeypair>)keypair
                                                    issuerID:(NSNumber * _Nullable)issuerID
                                                    fabricID:(NSNumber * _Nullable)fabricID
-                                                      error:(NSError * __autoreleasing _Nullable * _Nullable)error;
+                                                      error:(NSError * __autoreleasing _Nullable * _Nullable)error
+    API_AVAILABLE(TODOVERSION);
+
++ (nullable NSData *)generateRootCertificate:(id<MTRKeypair>)keypair
```

Comment:
Given we have significant amount of header documentation, I think it would be best to move the deprecated methods to the bottom of the file for clarity. I've also seen some API headers that put deprecated methods into a deprecated category.

Reply:
Yep, good idea. I will do the moving to the bottom, and will look into the deprecated category bit.